src/content/events_and_highlights.json

```diff
@@ -0,0 +1,128 @@
+{
+  "sections": [
+    {
+      "title": "Our Events and Highlights",
+      "content": [
+        {
+          "par": "AnitaB.org Open Source Events convenes members from all backgrounds, and levels of expertise throughout the year to bridge the diversity gap in Open Source development."
+        },
+        {
+          "par": "Our events aim to give members insights from industry experts and female-identifying leaders on a variety of topics promoting Open Source, and covering stages of Open Source development right from Introduction to Quality Assurance."
+        }
+      ]
+    },
+    {
+      "title": "Events",
+      "content": [
+        [
```

Comment:
this structure looks incorrect:
should be:
```
event:{ title:"bla bla bla", date:"bla bla bla", etc..}
```

Reply:
Is this schema okay? I also corrected the relative path to the image string. I think this is how it is needed in the .json file.
😅 

```{
      "Events": [
        {
          "Event Title": "..",
          "Event Date": "..",
          "Event Location": "...",
          "Event Timings": "...",
          "Event description": [
            {
              "par": "...."
            }
          ],
         "Event Highlights": "../src/assets/events_and_highlights/1.png",
          "Know more": "....."
       } 
  ]
}
```

Reply:
In keys we do not use uppercase and we use spaces. if `title` is nested in `event` there is no need to repeat `event` in key.